File: hw/tipan/U2/Hw18_JavaScript_animal/animal.js/src/Vaccine.js
const Animal = require('./Animal.js');

class Vaccine {
    constructor(id, description) {
        this.id = id;
        this.description = description;
    }

    toString() {
        return `Vaccine{id=${this.id}, description=${this.description}}`;
    }

    
    getId() {
        return this.id;
    }

    setId(id) {
        this.id = id;
    }

    getDescription() {
        return this.description;
    }

    setDescription(description) {
        this.description = description;
    }
}

module.exports = Vaccine;
  

File: ws/rojas/U2/WS15_Inheritance/ZooApp/src/main/java/ec/edu/espe/zoo/zooapp/model/Vaccine.java

package ec.edu.espe.zoo.zooapp.model;

/**
 *
 * @author USUARIO
 */
public class Vaccine {
    private int id;
    private String description;

    @Override
    public String toString() {
        return "Vaccine{" + "id=" + id + ", description=" + description + '}';
    }

    
    
    public Vaccine(int id, String description) {
        this.id = id;
        this.description = description;
    }

    
    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }
    
    
}
